docs/user/05-60-kyma-bindings.md

```diff
@@ -19,5 +19,5 @@ The Broker returns a kubeconfig file in the response body. The kubeconfig file c
 
 | Name                   | Default | Description                                                                                                                                                                                                                                                                                                                                                          |
 |------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **token_request**      | `false` | If set to `true`, the Broker returns a kubeconfig with a JWT token used as a user authentication mechanism. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such an approach allows for easily modifying the permissions granted to the kubeconfig. |
-| **expiration_seconds** | `600`   | Specifies the duration (in seconds) for which the generated kubeconfig is valid. If not provided, the default value of `600` seconds (10 minutes) is used.                                                                                                                                                                                                             |
\ No newline at end of file
+| **token_request**      | `false` | If set to `true`, the Broker returns a kubeconfig with a JWT token used as a user authentication mechanism. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such an approach allows for easily modifying the permissions granted to the kubeconfig.  Default value (false) generates the kubeconfig using Gardener's `shoots/adminkubeconfig`. |
```

Comment:
So eventually this is still `token_request`?
Not service_account? Or the discussion during refinement was about something different.

Reply:
jaroslaw-pieszka I will update it to `service_account` when the change is implemented. For now, I would suggest leaving it as it is, since that's the current state.